operator/src/lib.rs

```diff
@@ -40,6 +43,11 @@ use strum_macros::EnumIter;
 
 type KafkaReconcileResult = ReconcileResult<error::Error>;
 
+const FINALIZER_NAME: &str = "kafka.stackable.tech/cleanup";
+
+const APP_NAME: &str = "kafka";
+const MANAGED_BY: &str = "stackable-kafka";
```

Comment:
Add these to get_deletion_labels as well?

Reply:
Indeed. Done!